packages/react/src/components/Select/Select.tsx

```diff
@@ -163,7 +163,7 @@ const Select = (props: SelectProps) => {
     } else if (!arraysEqual(value, selectedValues)) {
       setSelectedValues(value ?? []);
     }
-  }, [value]);
+  }, [value, options]);
```

Comment:
I haven't looked to closely at the code in this component, but is this the correct way to solve this problem? `options` is not used in this effect, so this feels more like a workaround than a fix...? 🤔 

At a quick glance it looks like maybe the effect hook on [L110](https://github.com/digdir/designsystem/blob/cd1f43ccc78b77baf11a4f01221572ffd13470fe/packages/react/src/components/Select/Select.tsx#L110) needs to be updated with a options deps array.

Reply:
`options` is used in the sense that `findOptionFromValue` depends on `options`. 

There are many dependencies that are not included in the list, which causes it not to update when it should. There is actually a linting warning asking the list to be changed to:
`[findOptionFromValue, multiple, selectedValues, value]`

Which might also fix the problem.

Reply:
I changed it so that it uses all of the hook dependencies now, so no more linting warning

Reply:
Just to explain the logic behind the hook @mimarz is referring to: Having no dependencies makes it run on every render. However, there is an `if` statement at the start checking whether the following functions should run again. So basically, instead of comparing values in the dependency array, it just runs a normal `if` check, so it shouldn't have a significant impact on the performance. I understand that this is not a very clean way of doing it though, so I'll be happy if you have any suggestions on how to improve it.